contrib_bots/lib/EncryptBot.py

```diff
@@ -0,0 +1,54 @@
+def encrypt(text):
+    #This is where the actual ROT13 is applied
+    textlist = list(text)
+    newtext = ''
+    firsthalf = list('abcdefghijklmABCDEFGHIJKLM')
+    lasthalf = list('nopqrstuvwxyzNOPQRSTUVWXYZ')
+    for char in textlist:
+        if char in firsthalf:
+            newtext += lasthalf[firsthalf.index(char)]
```

Comment:
Since strings are immutable in Python; `+=` essentially means that you are destroying the old string and creating a new string. More Pythonic is to use `''.join` or a list comprehension.